Tests/FluentAssertions.Specs/Primitives/StringAssertionSpecs.EndWith.cs

```diff
@@ -83,6 +83,21 @@ public void When_string_ending_is_compared_with_string_that_is_longer_it_should_
                 "\"ABC\" is too short.");
         }
 
+        [Fact]
+        public void Throws_when_wrapped_into_an_assertion_scope_with_both_messages()
```

Comment:
🤔 I _think_ you intended to cover the scenario where both assertion messages should be included in the final message, but that has been changed recently. The last one isn't even evaluated anymore.

Reply:
Ah.. true, actually the test with both messages failed :) I just removed the second expected message part.. but I had overlooked, that then this test could be removed completely.

Reply:
Actually, this test is not completely useless: see the now missed lines (which previously weren't): https://coveralls.io/builds/70549123/source?filename=Src%2FFluentAssertions%2FPrimitives%2FStringEndStrategy.cs

Reply:
My point was that the title was a bit misleading.